sdks/java/io/kinesis/src/main/java/org/apache/beam/sdk/io/kinesis/SimplifiedKinesisClient.java

```diff
@@ -61,18 +69,33 @@
   private static final int PERIOD_GRANULARITY_IN_SECONDS = 60;
   private static final String SUM_STATISTIC = "Sum";
   private static final String STREAM_NAME_DIMENSION = "StreamName";
-  private static final int LIST_SHARDS_DESCRIBE_STREAM_MAX_ATTEMPTS = 10;
-  private static final Duration LIST_SHARDS_DESCRIBE_STREAM_INITIAL_BACKOFF =
+  private static final int LIST_SHARDS_MAX_RESULTS = 1_000;
+  private static final Duration
+      SPACING_FOR_TIMESTAMP_LIST_SHARDS_REQUEST_TO_NOT_EXCEED_TRIM_HORIZON =
```

Comment:
Can we name it with a shorter name?

Reply:
This name is so long to better describe what it does. if it would be shorter, it would be hard to describe what it does without adding comment to it. As such, I think it is better way to use the one I have proposed.

Reply:
Personally, I'd prefer to have a shorter name with a comment, which describes it in more details, but it's ok.